python/mxnet/optimizer.py

```diff
@@ -632,14 +632,76 @@ def update(self, index, weight, grad, state):
 class RMSProp(Optimizer):
     """RMSProp optimizer of Tieleman & Hinton, 2012,
 
+    This code follows the version in http://www.cs.toronto.edu/~tijmen/csc321/slides/lecture_slides_lec6.pdf
+    by Tieleman & Hinton, 2012
+
+    Parameters
+    ----------
+    learning_rate : float, optional
+        Step size.
+        Default value is set to 0.002.
+    gamma1: float, optional
+        decay factor of moving average for gradient^2.
+        Default value is set to 0.95.
+    wd : float, optional
+        L2 regularization coefficient add to all the weights
+    rescale_grad : float, optional
+        rescaling factor of gradient.
+    clip_gradient : float, optional
+        clip gradient in range [-clip_gradient, clip_gradient]
+    """
+    def __init__(self, learning_rate=0.002, gamma1=0.95,
+                 epsilon=1e-8, **kwargs):
```

Comment:
Yes. Change the python interface + the `.set_default`s in the C++ side so the docs will display correctly.
Shall we call it RMSPropCentered instead of RMSPropAlex? In fact, I feel that people do not use the centered version anymore. We may just add it for compatibility.